NFT/src/pages/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { FuelLogo } from "@/components/FuelLogo";
+import { HomeCard } from "@/components/HomeCard";
+import { Box, Grid, Stack } from "@mui/material";
+
+export default function Home() {
+  return (
+    <Stack>
+      <nav
+        className="flex justify-between items-center p-4 bg-black text-white gap-6 gradient-border
+            bg-gradient-to-b
+            from-zinc-900
+            to-zinc-950/80"
+      >
+        <Box
+          display="flex"
+          alignContent="center"
+          flexWrap="wrap"
+          justifyContent="center"
+          alignItems="center"
+          paddingTop="4px"
+          paddingBottom="4px"
+          width="stretch"
```

Comment:
Since we are also using tailwind, having these as tailwind classnames would make it neater